pkg/kubelet/userns/userns_manager.go

```diff
@@ -135,6 +133,11 @@ func MakeUserNsManager(kl userNsPodsManager) (*UsernsManager, error) {
 		return nil, err
 	}
 
+	userNsLength := kl.GetUserNsLength()
```

Comment:
Can't we add a field to the usernsManager struct with this, instead of asking the kubelet every time for this?

Reply:
done